coordinator/migrations/2024-05-23-000406_hodl_invoice_table/up.sql

```diff
@@ -0,0 +1,10 @@
+create table if not exists hodl_invoices
+(
+    id               SERIAL PRIMARY KEY       NOT NULL,
+    trader_pubkey    TEXT                     NOT NULL REFERENCES users (pubkey),
+    hash             TEXT                     NOT NULL,
+    amount_sats      BIGINT                   NOT NULL,
+    pre_image        TEXT,
+    timestamp        timestamp WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
nit: if we have a updated at this would make more sense to be named `created_at`.